### ISSUE TITLE
Question about Reid Models

### ISSUE THREAD

**Issue by ShinYoung-hwan** [mikel-brostrom/Yolov5_DeepSort_Pytorch#261]
Hello! I express my gratitude for your hard work.
These days, while analyzing DeepSort code, I found that you were trying to use the Reid models for DeepSort. Also, I found that osnet_x0_25 is used as a default in the code. 
In this part, I have some questions.

*Through this method, I would like to know how much the accuracy of DeepSort can be improved compared to the previous method.*
*Also, I want to know about your recommendation about Reid model to improve Deepsort result the most.*
*Finally, in the current code, only the pretrained model is used, but I want to know if there is any problem without using the finetuned result.*

Thank you.

**Comment by mikel-brostrom** [comment]
> Through this method, I would like to know how much the accuracy of DeepSort can be improved compared to the previous method.

I could measure a 2.5 MOTA increase only by changing model to the osnet_x0_25

> Also, I want to know about your recommendation about Reid model to improve Deepsort result the most

I still need to experiment with the ReID models to find the one that gives best overall accuracy for general object tracking (not necessarily only persons)

> Finally, in the current code, only the pretrained model is used, but I want to know if there is any problem without using the finetuned result.

Again, there is still a lot of experimentation to be done. Feel free to drop your results here or leave a PR with your findings!
